update documentation on automatic buffer refresh

zakora opened the pull request:
Github: magit/magit#3833

tarsius commented:
Thanks!

I was considering just enabling it by default, but lets leave that for another time.

You made the edit on Github, which I am forced to consider an unsuitable interface for editing code and creating commits.  Its the little things.  E.g. when using `git`, then every commit message ends with a newline character.  When using Github, then they don't unless you explicitly put it there; which nobody does.

We would like to properly attribute changes to people.  It may be possible to configure Github to do that, but the way you have configured it the commit is attribute to your Github alias and some dummy email.  We would like to use your real name and email; but won't force you to do so.

Please put an empty line between the paragraph and the code block.  Also

``` diff
-#+begin_src elisp
-#+BEGIN_SRC emacs-lisp
```

zakora commented:
Thank you for taking the time to look at this PR and explaining the details.

I updated the commit with magit so it should not contain any Github specific things now, and also changed the paragraph to match your suggestion.

Hope it's good now.
